core/src/main/scala/scalaz/EitherT.scala

```diff
@@ -228,7 +228,7 @@ object EitherT extends EitherTInstances {
   }
 
   def eitherTU[FAB, AB, A0, B0](fab: FAB)(
-    implicit u1: Unapply[Functor, FAB]{type A = AB}, u2: Unapply2[Bifunctor, AB]{type A = A0; type B = B0}, l: Leibniz.===[AB, A0 \/ B0])
```

Comment:
The unused `Unapply2` was there to guide type inference but it is probably no more necessary... ?